app/views/content/index.html.erb

```diff
@@ -20,28 +20,33 @@
             dates: [
               {
                 value: "past-30-days",
-                text: t("metrics.show.time_periods.past-30-days.leading"),
+                text: t("single_page.show.time_periods.past-30-days.leading"),
                 hint_text: "#{30.days.ago.strftime("%-d %B %Y")} to #{Date.yesterday.strftime("%-d %B %Y")}"
               },
               {
                 value: "last-month",
-                text: t("metrics.show.time_periods.last-month.leading"),
+                text: t("single_page.show.time_periods.last-month.leading"),
                 hint_text: "#{Date.today.last_month.beginning_of_month.strftime("%-d %B %Y")} to #{Date.today.last_month.end_of_month.strftime("%-d %B %Y")}"
               },
               {
                 value: "past-3-months",
-                text: t("metrics.show.time_periods.past-3-months.leading"),
+                text: t("single_page.show.time_periods.past-3-months.leading"),
                 hint_text: "#{(3.months.ago).strftime("%-d %B %Y")} to #{Date.yesterday.strftime("%-d %B %Y")}"
               },
               {
                 value: "past-6-months",
-                text: t("metrics.show.time_periods.past-6-months.leading"),
+                text: t("single_page.show.time_periods.past-6-months.leading"),
                 hint_text: "#{(6.months.ago).strftime("%-d %B %Y")} to #{Date.yesterday.strftime("%-d %B %Y")}"
               },
               {
                 value: "past-year",
-                text: t("metrics.show.time_periods.past-year.leading"),
+                text: t("single_page.show.time_periods.past-year.leading"),
                 hint_text: "#{(1.year.ago).strftime("%-d %B %Y")} to #{Date.yesterday.strftime("%-d %B %Y")}"
+              },
+              {
+                value: "past-2-years",
+                text: t("single_page.show.time_periods.past-2-years.leading"),
+                hint_text: "#{(2.years.ago).strftime("%-d %B %Y")} to #{Date.yesterday.strftime("%-d %B %Y")}"
```

Comment:
Is this change intentional? I remember something about this time range being removed, but I haven't checked.

Reply:
Ah well spotted, thanks Chris.